modules/serverless-logic-security-running-oauth-workflow-example.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-security
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-security-running-oauth-workflow-example_{context}"]
+= Running the OAuth2 orchestration workflow example locally
+
+The `serverless-workflow-oauth2-orchestration-quarkus` example application provides a hands-on demonstration of orchestrating a third-party secured service using a {ServerlessLogicProductName} workflow. You can clone, build, and run this example locally to observe OAuth 2.0 authentication, service orchestration, and workflow execution in action.
+
+.Prerequisites
+
+* You have installed Java 17 or later.
+* You have installed Apache Maven 3.9.3 or later.
+* You have installed Docker 20.10.7 or later.
+
+.Procedure
+
+. Clone the `kogito-examples` GitHub repository by using the following command:
++
+[source,terminal]
+----
+$ git clone https://github.com/kiegroup/kogito-examples.git
```

Comment:
```suggestion
$ git clone https://github.com/apache/incubator-kie-kogito-examples.git
```

The repo has changed once we moved to Apache.

Reply:
@gmunozfe since this is going to the product, can you verify if this example is still valid and working?